README.md

```diff
@@ -43,7 +43,10 @@
 
 - **Navigation bar with Logo**
 - **Hero section and Social media links**
-
+- **Work section: It contains my recent works**
+- **About me section which contains information about me including resume**
+- **Contact form**
+- **Footer**
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Kindly ad the valid information in your `README` file. Right now your project is not ready for deployment and you even didn't deploy it that's why it is important to mention in your `README` like *project is not deployed* instead of `misleading` the user with a *wrong* link.

Reply:
Okay, thank you very much.